modules/common-workflows/pages/workflow-install-from-raw-image.adoc

```diff
@@ -67,44 +66,34 @@ fdisk -l
 ----
 blockdev --getpbsz /dev/sda
 ----
-+
-
 +
 ----
 lsblk -o NAME,PHY-SEC /dev/sda
```

Comment:
If we skip device name, lsblk will show all and we can choose the right one. This command should be moved up to replace `fdisk -l` command.

```suggestion
lsblk -o NAME,PHY-SEC
```